tests/flow/test_persistency.py

```
@@ -407,3 +424,179 @@ def test07_persist_multiple_labels(self):
+        node_count = g.query("MATCH (a:A) RETURN count(a)").result_set[0][0]
+        self.env.assertEquals(node_count, 500)
+
+        # number of B nodes
+        node_count = g.query("MATCH (b:B) RETURN count(b)").result_set[0][0]
+        self.env.assertEquals(node_count, 500)
+
+        # number of edges
+        edge_count = g.query("MATCH ()-[e]->() RETURN count(e)").result_set[0][0]
+        self.env.assertEquals(node_count, 500)
+
+        edge_count = g.query("MATCH ()-[e:R]->() RETURN count(e)").result_set[0][0]
```

Ruff (0.12.2)

Local variable `edge_count` is assigned to but never used

Remove assignment to unused variable `edge_count`

(F841)